### DISCUSSION TITLE
Changing FastNoiseLite seed type from int to Long

### DISCUSSION THREAD

**Discussion by ghost** [godotengine/godot-proposals#9049] [General Discussions]
What is the cost of changing the type? will that be too much work? 

reason to change it is to increase the range and potential seeds. from 4 billion to 18 quintillion

**Comment by AThousandShips** [comment]
This will break compatibility, and unsure if the seed function for this allows 64 bits

Edit: this would require changes to the underlying library which only allows 32 bit seeds, so not just breaking compatibility but also not possible with the current library

**Comment by Calinou** [comment]
Could you describe the user case for needing more than ~4.2 billion different noise seeds? If you need greater randomness, why not randomize the noise offset as well?
